CHANGELOG.md

```diff
@@ -6,10 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Extending the adopted spec, each change should have a link to its corresponding pull request appended.
 
-## [Unreleased]
+## [v4.0.1] - 2019-08-23
 
 ### Added
 
+- Support for Forseti v2.19.1
```

Comment:
Should add the PR # for this one and make sure it links down below.

Reply:
Should also add [v4.0.1]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v4.0.0...v4.0.1 down below at Line 237.